Nest/00. Plan.md

```diff
@@ -0,0 +1,67 @@
+# 1 Introduction
+## 1.1 Package manager
+
+## 1.2 Packages and dependencies
+
+## 1.3 Repositories
+
+## 1.4 Package's syntax
+
+# 2 Nest
+## 2.1 Nest's options
+
+## 2.2 Nest commands
+### 2.2.1 Synchronizing the local cache
+### 2.2.2 Troubleshooting
+
+## 2.3 Installing and uninstalling a package
+### 2.3.1 Installing a package
+### 2.3.2 Uninstalling a package
+### 2.3.3 Troubleshooting
+
+## 2.4 Updating your system
+### 2.4.1 Troubleshooting
+
+## 2.5 Conclusion
+
+# 3 Some more advanced concepts
```

Comment:
I think a simpler sentence like `Advance concepts` would be better, (less words with the same meaning) what do you think?